source/common/network/io_socket_error_impl.cc

```diff
@@ -46,5 +46,15 @@ void IoSocketError::deleteIoError(Api::IoError* err) {
   }
 }
 
+inline IoSocketError* getIoSocketInvalidAddressInstance() {
+  static auto* instance = new IoSocketError(SOCKET_ERROR_NOT_SUP);
+  return instance;
+}
+
+Api::IoCallUint64Result IoSocketError::ioResultSocketInvalidAddress() {
```

Comment:
Is this unused now?

Reply:
I am calling this in checking sockAddr() in sendmsg()